src/main/java/study/racingcar/GameWinner.java

```diff
@@ -0,0 +1,35 @@
+package study.racingcar;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class GameWinner {
+
+    private int maxPosition;
+    private List<Car> carList;
+
+    public GameWinner(List<Car> carList) {
+        this.maxPosition = getMaxPosition(carList);
+        this.carList = carList;
+    }
+
+    public int getMaxPosition(List<Car> carList) {
+        List<Integer> positionList = new ArrayList<>();
+
+        for(Car car : carList) {
+            positionList.add(car.currentPosition());
+        }
+
+        return Collections.max(positionList);
```

Comment:
```suggestion
        return carList.stream()
                .map(Car::currentPosition)
                .max(Integer::compareTo)
                .orElse(0);
```
스트림을 사용하면 이런 식으로도 가능합니다.